bower.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "json-refs",
   "main": "browser/json-refs.js",
-  "version": "1.3.0",
```

Comment:
I've added https://github.com/mikaelbr/mversion/issues/40 so as to avoid this. So, for now, revert this particular aspect only in the PR?